Sources/UICollectionView+.swift

```diff
@@ -11,7 +11,7 @@ import UIKit
 public extension UICollectionView {
     public func register<T: UICollectionViewCell>(cellType: T.Type, bundle: Bundle? = nil) {
         let className = cellType.className
-        let nib = UINib(nibName: className, bundle: bundle)
+        let nib = UINib(nibName: className, bundle: bundle ?? Bundle(for: cellType))
```

Comment:
It's a good idea :)

But it has an unnecessary conditional branch, which affects performance.
I think default parameter is better than using `??` on runtime. don't you think so?

```swift
public func register<T: UICollectionViewCell>(cellType: T.Type, bundle: Bundle = Bundle(for: T.self)) {
```

Reply:
Thank you! Fixed:smile: